Sources/MapboxMaps/Style/Style.swift

```diff
@@ -165,25 +170,16 @@ public class Style {
      - Parameter type: The type of the source
      - Returns: The fully formed `source` object of type equal to `type`.
      - Throws: StyleError or type conversion errors
+     - Note: This method is marked as experimental. Annotate the import statement
+     for `MapboxMaps` with `@_spi(Experimental)` in order to use experimental methods.
      */
-    public func _source(withId id: String, type: Source.Type) throws  -> Source {
+    @_spi(Experimental) public func source(withId id: String, type: Source.Type) throws  -> Source {
```

Comment:
I don't think this needs to be experimental anymore. (same for the corresponding layer method above). We could actually knock out another issue related to these APIs at the same time by removing the variants that don't take a type parameter.

Reply:
That said, let's defer this feedback to our audit of all experimental APIs.